ext/hal/st/lib/sensor/vl53l0x/api/core/src/vl53l0x_api.c

```diff
@@ -382,10 +382,6 @@ VL53L0X_Error VL53L0X_DataInit(VL53L0X_DEV Dev)
 		0x01);
 #endif
 
-	/* Set I2C standard mode */
-	if (Status == VL53L0X_ERROR_NONE)
-		Status = VL53L0X_WrByte(Dev, 0x88, 0x00);
-

```

Comment:
this seems concerning, wondering why it had to be removed.

Reply:
Good hint. I had issue with this part while performing my developments. Now, the sensor is working OK with those lines. So I will reintroduce them in my next version.